cpg_workflows/jobs/joint_genotyping.py

```diff
@@ -234,6 +234,13 @@ def genomicsdb(
     # a significant amount of non-heap memory for native libraries.
     xms_gb = 8
     xmx_gb = 25
+    xmx_gb = config_retrieve(
+        [
+            'resource_overrides',
+            'genomicsdb_import_mem_gb',
+        ],
+        xmx_gb,
+    )
```

Comment:
```suggestion
    xmx_gb = config_retrieve(['resource_overrides',  'genomicsdb_import_mem_gb'], 25)
```

this is also a replacement for the line above

Reply:
Good note.
The default is `25 + 1 gb` requested memory, which Hail Batch has been rounding up to 32GB. 
Changed it to `xmx_gb + 4 gb` so it's a bit clearer, I don't imagine the actual requested memory for each VM will change though when this config option is unset.